src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -1018,12 +1272,6 @@ private enum RecoverAction
                                                           int level,
                                                           ref LocalGroupMemberInfo buf,
                                                           int totalEntries);
-        [DllImport("Netapi32.dll")]
-        public extern static int NetLocalGroupDelMembers([MarshalAs(UnmanagedType.LPWStr)] string serverName,
```

Comment:
its not being used anywhere. shall I revert this?

---
In reply to: [118028405](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118028405) [](ancestors = 118028405)